TILApp/Views/Shared/Components/CustomTextFieldView.swift

```diff
@@ -39,6 +31,23 @@ class CustomTextFieldView: UIView {
         set { textField.delegate = newValue }
     }
 
+    var readOnly: Bool = false {
+        didSet {
+            if readOnly {
+                textField.isUserInteractionEnabled = false
+                textField.textColor = .systemGray2
+            }
```

Comment:
readOnly를 설정했을 때만 아니라 다시 해제했을 때도 원래도 다시 동작하도록 해야 될 듯 하네요.

```suggestion
            textField.isUserInteractionEnabled = !readOnly
            textField.textColor = readOnly ? .systemGray2 : 원래색
```

이런 식으로...